views/remove-director-check-answers.html

```diff
@@ -40,6 +40,12 @@ <h2 class="govuk-heading-m">Company details</h2>
             }
           ]
         }) }}
+
```

Comment:
seems strange to have to add a new var name and not update directorName. Was directorName not used?

Reply:
Good spot, must have been left over from a copy/paste